src/main/java/com/rtb/projectmanagementtool/task/TaskController.java

```diff
@@ -0,0 +1,80 @@
+package com.rtb.projectmanagementtool.task;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key;
+import java.util.HashSet;
+
+/** Class controlling the TaskData object. */
+public final class TaskController {
+
+  private HashSet<TaskData> tasks;
+
+  public TaskController(HashSet<TaskData> tasks) {
+    this.tasks = tasks;
+  }
+
+  // Optional TODO: Add sort and quantity paramters.
+  public HashSet<TaskData> getAllTasks() {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    HashSet<TaskData> tasks = new HashSet<>();
+    Query query = new Query("Task");
+    PreparedQuery results = datastore.prepare(query);
+    for (Entity entity : results.asIterable()) {
+      // Get task entity data
+      long taskID = (long) entity.getKey().getId();
```

Comment:
I would move these to the Data layer, maybe pass an Entity as a param to its constructor.  It'll be like the Data object building itself based on a database entity.